integration/testrunner/db.go

```diff
@@ -66,13 +66,10 @@ func queryAuditLogs(ctx context.Context, tb testing.TB, query *bigquery.Query) [
 		if !ok {
 			tb.Fatalf("error converting query (%T) to string: %v", value[0], err)
 		}
-
+		tb.Logf("bq row is %s", value)
```

Comment:
Is this intended or a residual of your debug?

Reply:
Intended. During this debug, I found it is important to see what the bq row is becauase it could be different from the unmarshalled logentry. And this log will only appear when test fails.